docs/cicd_tools/common.md

```diff
@@ -0,0 +1,69 @@
+# Common module
+
+This module exposes helper functions that may be used (and shared) across all modules, or that don't
+serve a specific module's purpose to fit within it.
+
+## Definition
+
+The module ID is `common`
+
+All functions exposed by this module will use the namespaced prefix:
+
+```
+cicd::common::
+```
+
+## Usage
+
+Use the `common` id to load the module
+
+```
+CICD_TOOLS_URL="https://raw.githubusercontent.com/RedHatInsights/cicd-tools/main/src/bootstrap.sh"
+# shellcheck source=/dev/null
+source <(curl -sSL "$CICD_TOOLS_URL") common
+```
+
+This should load the function:
+
+```
+cicd::container::cmd
+```
+
+which serves as a wrapper to the container engine of choice. You should be able to safely replace
+your invocations to `docker` or `podman` commands with this function
+
+### Container engine detection and order choice
```

Comment:
It kinda feels like we transition into the container module docs here. Was that on purpose or a copy and paste oversight?